### ISSUE TITLE
Indeterminate checkboxes should not be "checked"

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#18295]
Current indeterminate/mixed checkboxes are announced as checked in a11y mode.  They should be considered mxied, or if this is not supported then (I think) "not checked" or false in mixed mode.

**Comment by HansMuller** [comment]
Could you update the description with a more comprehensive description of what's wrong?

**Comment by goderbauer** [comment]
Is this the same as https://github.com/flutter/flutter/issues/14677?

**Comment by jonahwilliams** [comment]
ahh yup, closing in favor of flutter/flutter#14677
